### PR TITLE
Updated source and conform for Carlton, MN

### PR DESCRIPTION
**Pull request by mmdolbow**
Got a better source for Carlton. I'm certain I don't have the regex quite right for the city and region tags because the typical contents are "<city> MN" - only a space between the city name and the state, no comma like the regex I have here. (I'm crummy at regex and trying to learn more.)

### CODE REVIEW

**Comment by trescube** [comment]
The `PHYSCITY` field doesn't contain commas though your regex is expecting them.  Instead of `^([^,]+)`, you can probably use `^(.*) MN$` for `city` and `^.* (mn)$` for `region`.

**Comment by mmdolbow** [comment]
Thanks for the tips, I'll try that!

File: sources/us/mn/carlton.json [hunk 1]
```diff
@@ -9,11 +9,31 @@
...
+        "street": {
+            "function": "regexp",
+            "field": "PHYSADDR",
+            "pattern": "^(?:[0-9]+) (.*)",
+            "replace": "$1"
+        },
+        "city": {
+            "function": "regexp",
+            "field": "PHYSCITY",
+            "pattern": "^(.*) MN$"
```
Comment (trescube) [inline sources/us/mn/carlton.json]: Sorry, I forgot to mention that you'll have to include `"replace": "$1"` in both `city` and `region` to replace then entire value with the first captured group.  It's very close, but the `city` and `region` fields in the output are blank.

File: sources/us/mn/carlton.json [hunk 2]
```diff
@@ -9,11 +9,31 @@
...
+        },
+        "region": {
+            "function": "regexp",
+            "field": "PHYSCITY",
+            "pattern": "^.* (MN)$"
+        },
```
Comment (trescube) [inline sources/us/mn/carlton.json]: Add `"replace": "$1"` here, too.

**Comment by migurski** [comment]
I see some trailing spaces in the sample data, e.g. `"ANYTOWN MN       "`; something like `"^(.*) MN\\s*$"` may fix it.

**Comment by mmdolbow** [comment]
OK, tried that fix.

**Comment by iandees** [comment]
I'm going to close this as these two sources changes appear to have been added to the master branch already.

**Pull request closed without merging**